standards/technical-approach.md

```diff
@@ -1,12 +1,30 @@
 # Technical Approach Documents
 
-Technical Approach Documents (TADs) serve to help the engineering team document solutions to feature development, share ideas between teams and help highlight potential issues with tasks before they are encountered during the development process.
+Technical Approach Documents (TADs) serve to help the engineering team document solutions to feature development, share ideas between teams, and help highlight potential issues with tasks before they are encountered during the development process.
 
-TADs should strive to describe the whats, hows and whys of the technical problem that is trying to be solved. It should strive to understand the problem at hand in a comprehensive way and explore any challenges so that engineers can be confident in their implementations of solutions. In short writing a TAD should answer any technical questions prior to the start of actual development work, freeing up developers to focus on implementation once the major questions have been resolved.
+TADs should strive to describe the whats, hows and whys of the technical problem that is trying to be solved. It should strive to understand the problem at hand in a comprehensive way and explore any challenges so that engineers can be confident in their implementations of solutions. In short, writing a TAD should answer any technical questions prior to the start of actual development work, freeing up developers to focus on implementation once the major questions have been resolved.
+
```

Comment:
I think the first two paragraphs could be a little more concise and direct about what the document is. What do you think? Maybe something along the lines of:

"Technical Approach Documents (TADs) allow engineers to explore multiple possible solutions to a technical problem before development begins, uncovering potential issues and ultimately finding a recommended approach through the process."

Reply:
Agreed, this is better. Thank you.